http/http-api/src/main/java/com/okta/commons/http/MimeType.java

```diff
@@ -422,7 +422,9 @@ private boolean parametersAreEqual(MimeType other) {
             return false;
         }
 
-        for (String key : this.parameters.keySet()) {
+        for (Map.Entry<String, String> entry : this.parameters.entrySet()) {
```

Comment:
Fixes below Findbugs error:

`com.okta.commons.http.MimeType.parametersAreEqual(MimeType) makes inefficient use of keySet iterator instead of entrySet iterator [com.okta.commons.http.MimeType] At MimeType.java:[line 435] WMI_WRONG_MAP_ITERATOR
`

Ref: https://stackoverflow.com/questions/12639259/findbugs-warning-inefficient-use-of-keyset-iterator-instead-of-entryset-iterato